tests/Rule/MimeTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator\Tests\Rule;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\UploadedFileInterface;
+use Yiisoft\Validator\Rule\Mime;
+
+/**
+ * @group validators
+ */
+class MimeTest extends TestCase
+{
+    public function testValidateString()
+    {
+        $val = Mime::rule(['image/jpeg']);
```

Comment:
```suggestion
        $value = Mime::rule(['image/jpeg']);
```

Don't use trim name, see https://github.com/yiisoft/docs/blob/master/010-code-style.md#names